src/doc/examples/ol-tracker.js

```diff
@@ -21,6 +21,7 @@ export default () => {
   const tracker = new TrajservLayer({
     url: 'https://api.geops.io/tracker/v1',
     apiKey: window.apiKey,
+    useDelayStyle: true,
```

Comment:
Are you sure of this?

Reply:
Indeed, this is leftover from test, good catch